keras_retinanet/models/__init__.py

```diff
@@ -0,0 +1,58 @@
+def custom_objects(backbone):
```

Comment:
It's good to factor out these operations from the scripts.

If we change this to objects per backbone with these operations it becomes a lot easier to add new backbones though. It would also be a good first step towards backbone plugins. Can be done in a future PR.

Reply:
I agree, let's do this in a different PR.